includes/helpers.php

```diff
@@ -5,3 +5,20 @@ function ctype_alnum( $text ){
 		return ( preg_match('~^[0-9a-z]*$~iu', $text ) > 0 );
 	}
 }
+
+if ( ! function_exists( 'update_entry_meta' ) ) {
+
+	/**
+	 * Update Entry meta
+	 *
+	 * Allows a meta to be altered before posting to Eloqua.
+	 *
+	 * @param  string $key  The key of the metadata.
+	 * @param  string $lead The value of the metadata.
+	 * @param  object $form The form that the metadata belongs to.
+	 * @return string       Returns the value of the entry meta.
+	 */
+	function update_entry_meta( $key, $lead, $form ) {
```

Comment:
I did a bit more research and it looks like the latest version uses the prefered Gravity Form function `gform_update_meta`. Are you finding calls to `update_entry_meta` in the plugin somewhere? If not, this is probably not necessary.